mhs/common/mhs_common/messages/ebxml_request_envelope.py

```diff
@@ -9,6 +9,7 @@
 import email.policy
 from typing import Dict, Tuple, Union, List, Sequence, Generator
 from xml.etree.ElementTree import Element
+from xml.etree import ElementTree
```

Comment:
no conflicts with:
'from defusedxml import ElementTree' 
from line 15 ?

Reply:
good point
it should be here